packages/esm-patient-forms-app/src/clinical-form-action-button.component.tsx

```diff
@@ -25,8 +26,14 @@ const ClinicalFormActionButton: React.FC = () => {
         kind="ghost"
         className={`${styles.container} ${isActiveWorkspace ? styles.active : ''}`}
         tabIndex={0}
-        disabled={isFormOpen}
-        onClick={launchFormsWorkspace}
+        onClick={
+          isFormOpen
+            ? () =>
+                launchPatientWorkspace('patient-form-entry-workspace', {
+                  workspaceTitle: recentlyOpenedForm?.additionalProps?.['workspaceTitle'],
+                })
+            : launchFormsWorkspace
+        }
```

Comment:
Thanks @denniskigen for the suggestions, for the above I did a little modification of including the else part because without it, it runs `launcFormsWorkspace` as the default.